docs/src/main/java-jdk21/docs/actors/classical/OptimizedActorWithJava21.java

```diff
@@ -0,0 +1,38 @@
+package docs.actors.classical;
```

Comment:
-1 on the dir rename - we should keep the existing name style

https://github.com/apache/incubator-pekko/blob/a2b5b5df22ec0820691bd84c358fb4352b97cbd0/project/Jdk9.scala#L26

Reply:
-1 for what? How can you compile a Java 21 code with Java 8/11/17?
The `"java-jdk9` file is still there.

BTW, I can't upgrade because of GFW, you can check it now.

Reply:
I see, I just upgraded, and the networking issue makes it very ...